pages/documentation/entropy/solidity-sdk.mdx

```diff
@@ -0,0 +1,91 @@
+# Pyth Entropy Solidity SDK
+
+The Pyth Entropy Solidity SDK allows you to generate secure random numbers on the blockchain by
+interacting with the Pyth Entropy protocol.
+The SDK can be used for any application that requires random numbers, such as NFT mints, gaming, and more.
+
+## Install
+
+```shell
```

Comment:
Can you add the copy attribute to the snippets were copy pasting makes sense like this one?
It would look like this:
```
```shell copy
```

Reply:
yeah sure